lightly/api/api_workflow_compute_worker.py

```diff
@@ -296,11 +297,25 @@ def schedule_compute_worker_run(
         """
         if runs_on is None:
             runs_on = []
-        config_id = self.create_compute_worker_config(
-            worker_config=worker_config,
-            lightly_config=lightly_config,
-            selection_config=selection_config,
-        )
+        try:
+            config_id = self.create_compute_worker_config(
+                worker_config=worker_config,
+                lightly_config=lightly_config,
+                selection_config=selection_config,
+            )
+        except ApiException as e:
+            eb = json.loads(e.body)
```

Comment:
maybe try catch this as well in the case it does not come in the expected form somehow? E.g when there is a ECONNRESET or other error?

Reply:
I guess the main point here is just to improve the handling of well-understood exceptions. If a worker config fails API-side validation with a 4xx return code, we can leave out all the HTTP headers and context, and the end of the stack trace handling the request.

We can cut to the chase and succinctly tell the user their config problem at this point. (Though still as an exception, for traceability & handling in any user application.)

But with an unexpected exception or something that isn't the user's fault (e.g. a 5xx), it's better just to dump the http headers and context and all information available to allow better debugging/ bug reports/ support requests. So I would just leave it like this.